lib/commands.sh

```
@@ -328,64 +366,412 @@ cmd_policy() {
+        else
+          local rows=""
+          rows+=$'Source\t'"${POLICY_SOURCE:-default}"$'\n'
+          rows+=$'Require Signed\t'"$signed_status"$'\n'
+          rows+=$'Allowed Authors\t'"${ALLOWED_AUTHORS_EFFECTIVE:-<none>}"$'\n'
+          rows+=$'Allowed Signers File\t'"${SIGNERS_FILE_EFFECTIVE:-<none>}"$'\n'
+          rows+=$'Notes Ref\t'"${NOTES_REF:-refs/_shiplog/notes/logs}"$'\n'
+          local raw_policy; raw_policy=$(load_raw_policy)
+          if [ -n "$raw_policy" ]; then
+            printf '%s\n' "$raw_policy" | jq -r '(.deployment_requirements // {}) | to_entries | map(select(.value.require_signed != null)) | .[] | "\(.key)\t\(.value.require_signed)"' 2>/dev/null | while IFS=$'\t' read -r env_name env_req; do
+              [ -z "$env_name" ] && continue
+              rows+=$'Require Signed ('"$env_name"$')\t'"$env_req"$'\n'
```

Shellcheck (0.11.0)

[info] Modification of rows is local (to subshell caused by pipeline).

(SC2030)